### PR TITLE
Greets user in Russian.

### PR DESCRIPTION
**Pull request by zoverlvx**

### CODE REVIEW

File: index.mjs [hunk 1]
```diff
@@ -4,7 +4,11 @@ export function makeGreeter(language) {
             return "\u00A1Hola" + " " + name + "!";
         };
     }
-
+    if (language === "Russian") {
+        return function (name = "\u043c\u0438\u0440") {
```
Comment (visaudi) [inline index.mjs]: мир or "\u043c\u0438\u0440" can be pulled into a variable.

File: index.mjs [hunk 2]
```diff
@@ -4,7 +4,11 @@ export function makeGreeter(language) {
...
+            return "\u041F\u0440\u0438\u0432\u0435\u0442\u0020" + name + "!";
```
Comment (visaudi) [inline index.mjs]: Привет or "\u041F\u0440\u0438\u0432\u0435\u0442\u0020" can also be pulled into a variable.

File: test.mjs [hunk 3]
```diff
@@ -14,6 +15,8 @@ assert.equal(
     makeSpanishGreeting("Andr\u00E9s"),
     "\u00A1Hola Andr\u00E9s!"
 );
+assert.equal(makeRussianGreeting(), "Привет мир!");
```
Comment (visaudi) [inline test.mjs]: `"Привет мир!"` should be a separate explicit UTF-8 escape character series that is stored in one variable.

File: index.mjs [hunk 4]
```diff
@@ -1,12 +1,24 @@
+const dictionary = [
```
Comment (zoverlvx) [inline index.mjs]: I have a more OO approach:

```
utf8Dictionary = {
    spanish: {
        upsideDownExclamation: ""
    },
    russian: {
        hello: "",
        world: ""
    }
};
```

Comment (visaudi) [reply index.mjs]: This is good. Let's also quote`"Spanish"` and `"Russian"`.

**Pull request merged**